content/hi/docs/setup/production-environment/container-runtimes.md

```diff
@@ -0,0 +1,403 @@
+[Install Docker Engine](https://docs.docker.com/engine/install/#server).
+
+2. कंटेनर को कॉन्फ़िगर करें:
+
+   ```shell
+   sudo mkdir -p /etc/containerd
+   containerd config default | sudo tee /etc/containerd/config.toml
+   ```
+
+3. कंटेनर को पुनरारंभ करें:
+
+   ```shell
+   sudo systemctl restart containerd
+   ```
+
+{{% /tab %}}
+{{% tab name="Windows (PowerShell)" %}}
+
+Powershell सत्र प्रारंभ करें, `$Version` को वांछित संस्करण पर सेट करें (उदाहरण: `$Version=1.4.3`), और फिर निम्न आदेश चलाएँ: 
+
+1. कंटेनरड डाउनलोड करें: 
+
+   ```powershell
+   curl.exe -L https://github.com/containerd/containerd/releases/download/v$Version/containerd-$Version-windows-amd64.tar.gz -o containerd-windows-amd64.tar.gz
+   tar.exe xvf .\containerd-windows-amd64.tar.gz
+   ```
+
+2. एक्सट्रेक्ट एंड कॉन्फ़िगर:
+
+   ```powershell
+   Copy-Item -Path ".\bin\" -Destination "$Env:ProgramFiles\containerd" -Recurse -Force
+   cd $Env:ProgramFiles\containerd\
+   .\containerd.exe config default | Out-File config.toml -Encoding ascii
+
+   # Review the configuration. Depending on setup you may want to adjust:
+   # - the sandbox_image (Kubernetes pause image)
+   # - cni bin_dir and conf_dir locations
+   Get-Content config.toml
+
+   # (Optional - but highly recommended) Exclude containerd from Windows Defender Scans
+   Add-MpPreference -ExclusionProcess "$Env:ProgramFiles\containerd\containerd.exe"
+   ```
+
+3. कंटेनर शुरू करें:
+
+   ```powershell
+   .\containerd.exe --register-service
+   Start-Service containerd
+   ```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+#### `systemd` cgroup ड्राइवर {#containerd-systemd} का उपयोग करें
+
+`systemd` cgroup ड्राइवर {#containerd-systemd} का उपयोग करना
+`runc` के साथ `/etc/containerd/config.toml` में `systemd` cgroup ड्राइवर का उपयोग करने के लिए, सेट करें
+
+```
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
+  ...
+  [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc.options]
+    SystemdCgroup = true
+```
+
+यदि आप इस परिवर्तन को लागू करते हैं, तो सुनिश्चित करें कि आप फिर से कंटेन्डर को पुनरारंभ करें: 
+
+```shell
+sudo systemctl restart containerd
+```
+
+Kubeadm का उपयोग करते समय, मैन्युअल रूप से कॉन्फ़िगर करें
+[क्यूबलेट के लिए cgroup ड्राइवर](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#configure-cgroup-driver-used-by-kubelet-on-control-plane-node).
+
+### CRI-O
+
+इस खंड में CRI-O को कंटेनर रनटाइम के रूप में स्थापित करने के लिए आवश्यक जानकारी है।
+
+अपने सिस्टम पर CRI-O स्थापित करने के लिए निम्नलिखित कमांड का उपयोग करें: 
+
+{{< note >}}
+CRI-O के प्रमुख और छोटे संस्करणों को Kubernetes के प्रमुख और छोटे संस्करणों से मेल खाना चाहिए। 
+अधिक जानकारी के लिए देखें  [CRI-O compatibility matrix](https://github.com/cri-o/cri-o#compatibility-matrix-cri-o--kubernetes).
+{{< /note >}}
+
+पूर्वापेक्षाएँ स्थापित और कॉन्फ़िगर करें:
+
+```shell
+# बूटअप पर मॉड्यूल लोड क करने के लिए .conf फाइल बनाएं Cr
```

Comment:
```suggestion
# बूटअप पर मॉड्यूल लोड करने के लिए .conf फाइल बनाएं
```

Reply:
Hi @rajatgupta24 , thank you for mentioning the changes, will update it